lib/Backends/CPU/CPUFunction.h

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef GLOW_BACKENDS_CPU_CPUFUNCTION_H
+#define GLOW_BACKENDS_CPU_CPUFUNCTION_H
+
+#include "GlowJIT.h"
+
+#include "glow/Backends/CompiledFunction.h"
+
+namespace glow {
+
+/// A function compiled for the CPU using LLVM.
```

Comment:
Nit: `function` in the comment is a bit ambiguous, because LLVM has a concept of functions too. May be it should say "Glow IR function" or "compiled network model"?